config/sync/user.role.gp_supervisor_user.yml

```diff
@@ -38,6 +38,24 @@ permissions:
   - 'moderate content from needs_review to draft'
   - 'moderate content from needs_review to published'
   - 'moderate content from published to needs_review'
+  - 'revert application revisions'
```

Comment:
Do GP supervisors need revision/edit access to non-GP related content types?

Reply:
Nope!